src/java.desktop/share/classes/sun/awt/OSInfo.java

```diff
@@ -58,11 +61,14 @@ and so the method getWindowsVersion() will return the constant for known OS.
     public static final WindowsVersion WINDOWS_VISTA = new WindowsVersion(6, 0);
     public static final WindowsVersion WINDOWS_7 = new WindowsVersion(6, 1);
```

Comment:
The  windows version map looks like it hasn't had any love in a long time.
Updating it is VERY out of scope for this fix as it needs some thought in whether to remove old entries
and what to do about replacing them so I filed https://bugs.openjdk.org/browse/JDK-8305777